vm/vm.go

```diff
@@ -150,6 +128,40 @@ func (vm *VM) newThread() (t Thread) {
 	return
 }
 
+// vm.assignLibPath looks up and assigns vm.libPath
+func (vm *VM) assignLibPath() (err error) {
+	if DefaultLibPath != "" {
+		vm.libPath = DefaultLibPath
+		return
+	}
+
+	gobyRoot := os.Getenv("GOBY_ROOT")
+
+	if len(gobyRoot) == 0 {
+		// fallback to homebrew's path
+		gobyRoot = fmt.Sprintf("/usr/local/Cellar/goby/%s", Version)
```

Comment:
I'd use `brew` commands to get the formula (I'm not familiar with Homebrew terminology) rather than using a hardcoded path, as it's a more generic (and robust) approach.

Reply:
This is a good point. But I think that should be addressed in a different PR. Here's the issue for it https://github.com/goby-lang/goby/issues/846